explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -361,12 +363,28 @@ object ObsTabTiles:
             )
 
           val sequenceTile =
-            SequenceEditorTile.sequenceTile(
-              props.programId,
-              props.obsId,
-              asterismIds.get.toList,
-              itc.toOption.flatten.map(_.snPerClass).getOrElse(Map.empty),
-              sequenceChanged
+            SequenceEditorTile.sequenceTile(renderInTitle =>
+              React.Fragment(
+                renderInTitle {
+                  val programTimeCharge = props.observation.get.execution.timeCharge.program
+                  props.observation.get.executionTime
+                    .map { planned =>
+                      val total = programTimeCharge +| planned
+                      <.span(
+                        <.span(^.fontWeight := "bold", total.toHoursMinutes),
```

Comment:
`^.fontWeitght.bold`. Can we move styling to a class?

Reply:
Moved it to a class